HousingRepairsOnlineApi/UseCases/IRetrieveAvailableAppointmentsUseCase.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 using HACT.Dtos;
+using HousingRepairsOnlineApi.Domain;
 
 namespace HousingRepairsOnlineApi.UseCases
 {
     public interface IRetrieveAvailableAppointmentsUseCase
     {
-        public Task<IEnumerable<Appointment>> Execute(string repairLocation, string repairProblem, string repairIssue, string uprn);
+        public Task<List<AppointmentTime>> Execute(string repairLocation, string repairProblem, string repairIssue,
+            string uprn);
```

Comment:
Please rename `uprn` to `locationId` to avoid confusion about the value of the field.